model/accounts_mgmt/v1/account_type.model

```diff
@@ -26,5 +26,5 @@ class Account {
 	UpdatedAt Date
 	ServiceAccount Boolean
 	Organization Organization
-	Labels []Label
+	Labels []AccountLabel
```

Comment:
Does the `Account` model have the `Labels` directly or as a link? In the `clusters_mgmt` API, when a resource contains a list of another resource, we typically set it as a link. For example:
```
{
  "kind": "Account",
  "href": "/api/accounts_mgmt/v1/account/###",
  "labels": {
    "kind": "AccountLabelsLink",
    "href": "/api/accounts_mgmt/v1/account/###/account_labels"
  }
}
```
In most cases we don't expose the `"items"` directly on the parent resource and let the client request those manually as needed.

If this is (or should be) the case, then this line should change to
```
link Labels []AccountLabel
```

Reply:
Labels are direct part of Account/Organization/Subscription

Reply:
Got it, thanks